### ISSUE TITLE
Please keep tab order on restore

### ISSUE THREAD

**Issue by bsw11** [nickbaum/simple-window-saver#14]
Is there a way to maintain the tab order on restore?  This would be very helpful.

**Comment by nickbaum** [comment]
Hey, thanks for the report.

Tab order should be maintained on restore. If this isn't the case, can you give me some more details on exactly the steps you take and what OS and browser version you're using?

**Comment by nickbaum** [comment]
Closing this since I haven't been able to reproduce and haven't heard any further reports.
